### PR TITLE
Add NFTs page

### PR DESCRIPTION
**Pull request by serjonya-trili**
## Proposed changes

[Task link](https://app.asana.com/0/1207041888417083/1207650864030724/f)

## Types of changes

- [ ] Bugfix
- [x] New feature
- [x] Refactor
- [ ] Breaking change
- [ ] UI fix

## Screenshots

Screenshot 2024-08-05 at 07 56 17
Screenshot 2024-08-05 at 07 56 25

Screenshot 2024-08-05 at 16 27 22
Screenshot 2024-08-05 at 16 27 30

## Checklist

- [x] Tests that prove my fix is effective or that my feature works have been added
- [ ] Documentation has been added (if appropriate)
- [ ] Screenshots are added (if any UI changes have been made)
- [ ] All TODOs have a corresponding task created (and the link is attached to it)

### CODE REVIEW

File: apps/web/src/views/NFTs/NFTs.tsx [hunk 1]
```diff
@@ -1,9 +1,52 @@
...
-);
+export const NFTs = () => {
+  const color = useColor();
+  const account = useCurrentAccount()!;
+  const nfts = sortedByLastUpdate(useGetAccountNFTs()(account.address.pkh));
+  const totalCount = nfts.reduce((acc, nft) => acc.plus(nft.balance), BigNumber(0)).toNumber();
+
+  let gridTemplateColumns = "repeat(auto-fit, minmax(min(100%/2, max(157px, 100%/4)), 1fr))";
+  if (nfts.length < 3) {
+    gridTemplateColumns = `repeat(auto-fit, min(100% / ${nfts.length} - 6px, 100%))`;
```
Comment (OKendigelyan) [inline apps/web/src/views/NFTs/NFTs.tsx:19]: What do we calculating here `${nfts.length} - 6px`?

File: apps/web/src/views/NFTs/NFTCard.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          />
+        </Center>
+      </CardBody>
+      <Box overflow="hidden" data-testid="name">
+        <Heading
+          overflow="hidden"
+          color={color("900")}
+          whiteSpace="nowrap"
+          textOverflow="ellipsis"
+          size="sm"
```
Comment (OKendigelyan) [inline apps/web/src/views/NFTs/NFTCard.tsx:39]: According figma fontSize should be **14px**
heading sm equals **16px**

File: apps/web/src/views/NFTs/NFTCard.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import { useColor } from "../../styles/useColor";
+
+export const NFTCard = ({ nft }: { nft: NFTBalance }) => {
+  const color = useColor();
+  const url = getIPFSurl(thumbnailUri(nft));
+  const fallbackUrl = getIPFSurl(nft.displayUri);
+  const name = nft.metadata.name;
+
+  return (
+    <Card
```
Comment (OKendigelyan) [inline apps/web/src/views/NFTs/NFTCard.tsx:14]: Hover is missing

image

File: apps/web/src/views/NFTs/NFTs.tsx (hunk 1 above)
Comment (serjonya-trili) [reply apps/web/src/views/NFTs/NFTs.tsx:19]: without it the 2nd image doesn't fit and wraps around to the 2nd row
Screenshot 2024-08-05 at 16 25 42

**Review by OKendigelyan — APPROVED** [review]

**Pull request merged**